### PR TITLE
DEVELOPER-1340 - Create a new Faraday Middleware component in Aweplug…

### PR DESCRIPTION
**Pull request by Dantheman720**
… that handles response status codes

### CODE REVIEW

**Comment by LightGuard** [comment]
retest this please

File: Gemfile.lock [hunk 1]
```diff
@@ -1,29 +1,26 @@
 GIT
   remote: git://github.com/LightGuard/awestruct.git
-  revision: 05bf2c7fc4027c00115b6bc2e9ff0ffcf3eacc8e
+  revision: 5dbe512680b7aa094598ecdcbb6e52d3e75aa32f
```
Comment (LightGuard) [inline Gemfile.lock]: Uh, yeah, don't use my version, use awestruct/awestruct please.

Comment (Dantheman720) [inline Gemfile.lock]: Hrm, I didn't edit that on the gemfile.  I will add awestruct/awestruct to the Gemfile.

**Comment by Dantheman720** [comment]
retest this please

**Comment by LightGuard** [comment]
Looks like the latest is an issue with books.

**Comment by Dantheman720** [comment]
Agreed.  I'm going to work on that books spreadsheet so that this problem will be solved...probably

**Comment by paulrobinson** [comment]
retest this please

File: Gemfile.lock (hunk 1 above)
Comment (paulrobinson) [inline Gemfile.lock]: You need to run `bundle install` and then add the new `Gemfile.lock`.

Comment (LightGuard) [inline Gemfile.lock]: Why are you using my fork?

On Tuesday, July 14, 2015, Paul Robinson <notifications@github.com> wrote:

> In Gemfile.lock
> <https://github.com/jboss-developer/www.jboss.org/pull/920#discussion_r34546615>
> :
>
> > @@ -1,29 +1,26 @@
> >  GIT
> >    remote: git://github.com/LightGuard/awestruct.git
> > -  revision: 05bf2c7fc4027c00115b6bc2e9ff0ffcf3eacc8e
> > +  revision: 5dbe512680b7aa094598ecdcbb6e52d3e75aa32f
>
> You need to run bundle install and then add the new Gemfile.lock.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jboss-developer/www.jboss.org/pull/920/files#r34546615>
> .
>


-- 
Sent from Gmail Mobile

**Comment by Dantheman720** [comment]
@paulrobinson I can't get this to work using bundle install.  It tells me that 'bundle update' will only use the gems from my gemfile.  Can I use that instead?  Otherwise I get dependency issues from Awestruct.

**Pull request merged**